### ISSUE TITLE
Running into errors

### ISSUE THREAD

**Issue by Synchrololled69** [nwithan8/Tauticord#11]
So i'm both trying to run it as docker and manually but it both ways runs into issues, for docker it just dies and I have no folder to attach to it / cannot find the folder it builds to. and when running it on my windows machine it spits out it can't find the ID of my libraries, neither tautulli named nor plex named. 

I have filled in the api, ip, discord server, channel and such it does create them but it doesn't seem to be working properly. 

any chance I could get some assistance? 
![image](https://user-images.githubusercontent.com/22560492/116825078-dd82b000-ab8d-11eb-9490-ff74e0150e93.png)

**Comment by PillowEmperor** [comment]
I've now also tried it on an ubuntu machine, spits out the same error but not the PyNaCl one.

**Comment by PillowEmperor** [comment]
Yeah I would love to get it working aswell, got a few friends who use my plex and got a discord so that they can add movies and series but would be nice for them to see the activity aswell in discord

**Comment by Shayne55434** [comment]
> 
> 
> Yeah I would love to get it working aswell, got a few friends who use my plex and got a discord so that they can add movies and series but would be nice for them to see the activity aswell in discord

I've got a few Tautulli/Discord scripts I use, currently. Maybe they will be useful to you in the meantime.
https://github.com/Shayne55434/Tautulli2Discord

**Comment by PillowEmperor** [comment]
> > Yeah I would love to get it working aswell, got a few friends who use my plex and got a discord so that they can add movies and series but would be nice for them to see the activity aswell in discord
> 
> I've got a few Tautulli/Discord scripts I use, currently. Maybe they will be useful to you in the meantime.
> https://github.com/Shayne55434/Tautulli2Discord

I'll need to give it quite the workaround and run a VM with windows then but yeah I'll probably be able to make this work for now, thanks!
